### ISSUE TITLE
data.json

### ISSUE THREAD

**Issue by RS-labhub** [swapagarwal/swag-for-dev#1198]
I wanted to add Coding Ninjas swag opportunity. Please review on this.

**Comment by BandhiyaHardik** [comment]
Hey @RS-labhub,

I noticed a way to improve things. When creating issues or PRs, clear titles are a big help.

I came across this while checking out an issue. Having straightforward titles makes life easier for everyone. It helps maintainers and sets you up for smoother interactions in the future.

Just wanted to pass this tip along.

**Comment by RS-labhub** [comment]
> Hey @RS-labhub,
> 
> I noticed a way to improve things. When creating issues or PRs, clear titles are a big help.
> 
> I came across this while checking out an issue. Having straightforward titles makes life easier for everyone. It helps maintainers and sets you up for smoother interactions in the future.
> 
> Just wanted to pass this tip along.

Yeah okay.. Actually I'm new to open source and contributing for the 1st time...

**Comment by BandhiyaHardik** [comment]
@RS-labhub  I am also new to OpenSource just like you.
